torch/_inductor/config.py

```diff
@@ -577,6 +577,9 @@ class cuda:
     # are enabled for the CUTLASS backend.
     cutlass_only_evt_capable_ops: bool = False
 
+    # Minimum of M*N*N to consider the CUTLASS backend for GEMM ops.
+    cutlass_backend_min_gemm_size: int = 1
```

Comment:
Specifying the GEMM size threshold with a single number sounds a bit too simplistic. E.g., there can be tall or thin GEMMs where CUTLASS may outperform Triton, but the `M * N * K` would be as small as a moderately-sized square GEMM. Can't suggest an immediate alternative, though, as no total ordering in GEMM sizes.